pom.xml

```diff
@@ -7,83 +7,15 @@
   </parent>
 
   <groupId>org.nuxeo.ecm.platform</groupId>
```

Comment:
No, I can't remove this one. Nuxeo Agenda belongs to `org.nuxeo.ecm.platform`, which is different from its parent _nuxeo-addons-parent_. This line is different from the ones in module _core_ or module _jsf_.